index.html

```diff
@@ -51,6 +51,166 @@ <h2 class="color-green">I'm a software developer</h2>
+            </li>
+            <li class="list-item">
+              <div class="image-placeholder"></div>
+              <div class="image-desc">
+                <h2>Multi-Post Stories Gain+Glory</h2>
+                <div class="buttons">
+                  <div>Ruby on rails</div>
+                  <div>css</div>
+                  <div>JavScript</div>
+                  <div>html</div>
+                </div>
+                <div class="button-link">
+                  <button class="button" type="button">See Project</button>
+                </div>
+              </div>
+            </li>
+            <li class="list-item">
+              <div class="image-placeholder"></div>
+              <div class="image-desc">
+                <h2>Multi-Post Stories Gain+Glory</h2>
+                <div class="buttons">
+                  <div>Ruby on rails</div>
+                  <div>css</div>
+                  <div>JavScript</div>
+                  <div>html</div>
+                </div>
+                <div class="button-link">
+                  <button class="button" type="button">See Project</button>
+                </div>
+              </div>
+            </li>
+            <li class="list-item">
+              <div class="image-placeholder"></div>
+              <div class="image-desc">
+                <h2>Multi-Post Stories Gain+Glory</h2>
+                <div class="buttons">
+                  <div>Ruby on rails</div>
+                  <div>css</div>
+                  <div>JavScript</div>
+                  <div>html</div>
+                </div>
+                <div class="button-link">
+                  <button class="button" type="button">See Project</button>
+                </div>
+              </div>
+            </li>
+            <li class="list-item">
+              <div class="image-placeholder"></div>
+              <div class="image-desc">
+                <h2>Multi-Post Stories Gain+Glory</h2>
+                <div class="buttons">
+                  <div>Ruby on rails</div>
+                  <div>css</div>
+                  <div>JavScript</div>
+                  <div>html</div>
+                </div>
+                <div class="button-link">
+                  <button class="button" type="button">See Project</button>
+                </div>
+              </div>
+            </li>
+          </ul>
+        </div>
+      </section>
+
+      <div class="about-me">
+        <section class="about-container">
+          <header class="about-header">
+            <h1>About me</h1>
+          </header>
+          <div class="about-desc">
+            <p>
+              I can help you build a product , feature or website Look through
+              some of my work and experience! If you like what you see and have
+              a project you need coded, don’t hestiate to contact me.
+            </p>
+          </div>
+          <div class="resume-button">
+            <button type="button">Get My Resume</button>
```

Comment:
- [ ] Kindly add styles to this and all buttons that respond to the `:active` and `:hover` subclasses, so that it changes colors when you hover it with your mouse. This is listed in the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md)